sa/sa.go

```diff
@@ -1757,3 +1757,13 @@ func (ssa *SQLStorageAuthority) SerialExists(ctx context.Context, req *sapb.Seri
 	exists := !isNoRowsErr
 	return &sapb.Exists{Exists: &exists}, nil
 }
+
+func addKeyHash(db db.Inserter, cert *x509.Certificate) error {
+	h := sha256.Sum256(cert.RawSubjectPublicKeyInfo)
```

Comment:
Let's add a nil check on cert.RawSubjectPublicKeyInfo and return an error if it's nil.

Reply:
I'm like 99% sure there is no scenario where this can happen (if this assumption was invalidated a lot of our code would break, not just here).